crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -103,6 +103,26 @@ fn rmain() -> Result<(), Error> {
     let headless = env::var("NO_HEADLESS").is_err();
     let debug = env::var("WASM_BINDGEN_NO_DEBUG").is_err();
 
+    // Gracefully handle requests to execute only node or only web tests.
+    if env::var_os("WASM_BINDGEN_TEST_ONLY_NODE").is_some() {
+        if !node {
+            println!("this test suite is only configured to run in a browser, \
+                      but we're only testing node.js tests so skipping");
+            return Ok(());
+        }
+    }
+    if env::var_os("WASM_BINDGEN_TEST_ONLY_WEB").is_some() {
+        if node {
+            println!("this test suite is only configured to run in node.js, \
+                      but we're only running browser tests so skipping");
+            println!("");
+            println!("if you'd like to run the tests in a browser you can \
+                      use the `wasm_bindgen_test_configure!(run_in_browser);` \
```

Comment:
Might as well add the import and everything in a 4-space indented code block, since people will likely want to just copy paste this immediately.